src/stories/ButtonWithDropdownMenu.stories.tsx

```diff
@@ -11,23 +15,23 @@ const story = {
     component: ButtonWithDropdownMenu,
 };
 
-function Template(args) {
+function Template(props: ButtonWithDropdownMenuProps<unknown>) {
     // eslint-disable-next-line react/jsx-props-no-spreading
-    return <ButtonWithDropdownMenu {...args} />;
+    return <ButtonWithDropdownMenu {...props} />;
 }
 
 // Arguments can be passed to the component by binding
 // See: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
-const Default = Template.bind({});
+const Default: ButtonWithDropdownMenuStory = Template.bind({});
 Default.args = {
-    buttonText: 'Pay using Expensify',
+    customText: 'Pay using Expensify',
```

Comment:
@ruben-rebelo Why do you change here?

Reply:
@DylanDylann This was changed because the actual prop which sets the text of the button is now customText instead of buttonText.